src/nti/site/interfaces.py

```diff
@@ -124,3 +139,21 @@ def __call__(func, retries=0, sleep=None, site_names=(), side_effect_free=False,
 
         :return: The value returned by the first successful invocation of `func`.
         """
+
+class ISiteMapping(interface.Interface):
+    """
+    Maps a site name to an alternate site. Useful when we do not want full
+    fledged sites in the dataserver. Should only be used after checking for an
```

Comment:
We should try to avoid specific language about "dataserver" in favor of more generic descriptions.